big_tests/tests/graphql_account_SUITE.erl

```diff
@@ -401,6 +413,57 @@ admin_change_non_existing_user_password(Config) ->
     Resp3 = ban_user(?EMPTY_NAME_JID, NewPassword, Config),
     get_coercion_err_msg(Resp3).
 
+admin_import_users_cli(Config) ->
+    escalus:fresh_story(Config, [{alice, 1}], fun(_Alice) ->
+        % Non-existing file
+        Resp = import_users(<<"nonexisting.csv">>, Config),
+        ?assertEqual(<<"File not found">>, get_err_msg(Resp)),
+        % Summary
+        Path = filename:join(?config(mim_data_dir, Config), "users.csv"),
+        Path2 = replace_hosts_in_file(Path),
+        Resp2 = import_users(list_to_binary(Path2), Config),
+        Domain = domain_helper:domain(),
+        ?assertEqual(#{<<"status">> => <<"Completed">>,
+                       <<"created">> => [<<"john@", Domain/binary>>],
+                       <<"emptyPassword">> => [<<"elise@", Domain/binary>>],
+                       <<"existing">> => [<<"alice@", Domain/binary>>],
+                       <<"invalidJID">> => [<<",", Domain/binary, ",password">>],
+                       <<"invalidRecord">> => [<<"elise,elise,", Domain/binary, ",esile">>],
+                       <<"notAllowed">> => [<<"john@", Domain/binary>>]},
```

Comment:
Why is this one expected to be "not allowed"? It is seems identical to the "created" one.

Reply:
Also, I deleted this case because it wasn't working correctly for `riak` backend. The import creates a several processes to register users from the file. Unfortunately, `riak` allows to register two identical users when requests are done in the same time (other backends returns `exists`).

I will create a dedicated ticket for that issue.